docker-compose.yml

```diff
@@ -5,6 +5,12 @@ services:
   postgresql:
     image: postgres:12-alpine
     restart: unless-stopped
+    healthcheck:
+      test: ["CMD", "pg_isready"]
+      start_period: 20s
```

Comment:
`start_period` seems to have been removed:

```
root@beryjuorg-do:/srv/authentik# docker-compose up -d
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.postgresql.healthcheck value Additional properties are not allowed ('start_period' was unexpected)
services.redis.healthcheck value Additional properties are not allowed ('start_period' was unexpected)
```

Reply:
oh wait, it requires 3.4 of the compose file, feel free to bump it to `3.4` at the top